i18n.json

```diff
@@ -1,5 +1,5 @@
 {
-  "locales": ["en-us", "vi"],
+  "locales": ["en-us", "vi-vi"],
```

Comment:
I don't think `vi-vi` is in the standard.

Reply:
Every locale should follow the syntax `[language]-[country]` tho... But it works, no?

Reply:
Oh actually, it should be `vi-vn`

Reply:
According to this table https://docs.oracle.com/cd/E13214_01/wli/docs92/xref/xqisocodes.html

Reply:
Urghh, you sure? Technically we can set it as anything as we like for next-translate, but I'm following the `accept-language` in request header and `lang` tag in html.

Reply:
![image](https://user-images.githubusercontent.com/52666982/145920855-8cb8d3d3-b1fb-4841-b4d7-63324c3f16f7.png)

Reply:
You can check this by going to Chrome settings and adding more languages.

Reply:
MDN docs: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Accept-Language

```
<language>
A language tag (which is sometimes referred to as a "locale identifier"). This consists of a 2-3 letter base language tag that indicates a language, optionally followed by additional subtags separated by '-'. The most common extra information is the country or region variant (like 'en-US' or 'fr-CA') or the type of alphabet to use (like 'sr-Latn'). Other variants, like the type of orthography ('de-DE-1996'), are usually not used in the context of this header.
```

Reply:
So basically we can add anything extra after that for additional context

Reply:
Perhaps we should do `en-ca` to match the user demographic

Reply:
I guess `vi-vn` is fine.